README.md

```diff
@@ -25,4 +25,61 @@ To Pull and Run this code locally,
 1. Install PostgreSQL. Make sure to install pgAdmin with it. When prompted, set the username and password to postgres/postgres
 2. Create a "shopify" database from pgAdmin.
 3. Open and build the project in IntelliJ
-4. Use the "Run Mini Shopify" Run Configuration. When the application finishes starting up, it can be accessed at localhost:8080
\ No newline at end of file
+4. Use the "Run Mini Shopify" Run Configuration. When the application finishes starting up, it can be accessed at localhost:8080
+
+To host the front end,
+
+1. cd into shopify-client
+2. run `yarn install` or `npm install`
+3. run `yarn run serve` or `npm run serve`
```

Comment:
You should add instruction for how to access the front end.

4. Navigate to localhost:8081 in a JS enabled browser

or something like that.

Reply:
on top of this, it would be a good idea to point out that our project is designed to be used by modern browsers

Reply:
Or just say no IE.

Reply:
I mean the terminal tells you were to access it, if you have something else hosted on 8081 then it won't be 8081

Reply:
Fair enough, but they should have instruction about the JS enabled modern browser at least.

Reply:
I think it's safe to assume that they will be using modern browsers. I think Github has dropped support for IE. They won't be looking at this repo if they were using IE.

Reply:
In specifications for web apps, you should always mention if it's restricted to modern browsers or not.
If this is an actual product, it would be a valid concern.